button_res.py

```diff
@@ -0,0 +1,20 @@
+import tkinter as tk
+
+
+class App(tk.Tk):
+    def __init__(self):
+        super().__init__()
+        self.geometry("700x600")
+        self.title("Minesweeper")
+        but1 = Button_play()
```

Comment:
Краще уникати назв кнопок просто з індексами (but1, but2, ...).
Назва повинна давати розуміння, що ми зберігаємо в змінній
```suggestion
        play_button = Button_play()
```

Те саме з навзою файла, вона має бути зрозумілою і описувати, що у файлі `play_button.py` чи `button_play.py`